services/navigation/builtin/builtin.go

```diff
@@ -239,10 +267,11 @@ func (svc *builtIn) Reconfigure(ctx context.Context, deps resource.Dependencies,
 	svc.mode = navigation.ModeManual
 	svc.storeType = string(svcConfig.Store.Type)
 	svc.base = base1
-	svc.movementSensor = movementSensor
+	svc.mapType = mapType
 	svc.motion = motionSvc
 	svc.obstacles = newObstacles
 	svc.replanCostFactor = svcConfig.ReplanCostFactor
+	svc.vision = visionSvcs
```

Comment:
[note] This is the vision service to be used by explore. Do we want to use all of the ones given or just the first one?

Reply:
[note #2] the call to GetObjectPointClouds also requires a camera input, it is not attached to the vision service. We may need an extra input of cameraName.

Reply:
Could you explain this a little more?

Reply:
Here is a mock of what our random walk implementation will look like: https://github.com/jeremyrhyde/rdk/blob/acf0e59e25a6fbdf5f40f448da67aa12cf201a7f/services/navigation/builtin/explore.go#L57.

As you can see on line 57, `GetObjectPointCloud` is a function of a vision service and takes in a camera. We currently have no way in navigation to know which camera name should go here.